### DISCUSSION TITLE
thanks you for developing this, but I have a laptop with AMD and nVIDIA....

### DISCUSSION THREAD

**Discussion by freebrew** [sultim-t/xash-rt#61] [General]
The game and mod does not detect nVIDIA for DLSS not raytrracing as it is defaulting to use the smaller intergrated AMD iGPU on die instead of the deticated nVIDIA GPU.

Any advice?, I have uploaded the optional rt DLSS files in the appropriate folder and started the game with xash3d.exe executable.

Please advise

**Comment by iModes** [comment]
I think you have to set it in the Windows settings wich gpu will be used. Try to find: Settings  -> Display -> Graphics. There will be an list of all apps you are using and then you can manualy choose the xash3d.exe.
I hope it helps!

**Reply by palaashatri** [reply]
Or maybe they can force the game to render on the Nvidia GPU through the Nvidia Control Panel too?

**Reply by iModes** [reply]
Yes, you are right. Don't thinked about it.
